src/components/sections/IdentifiersSection/IdentifiersSection.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import RadioField from 'components/forms/RadioField';
+
+const options = [
+  { value: 'true', label: 'Yes' },
+  { value: 'false', label: 'No' }
+];
+
+function IdentifiersSection() {
+  return (
+    <RadioField
+      label="Is the case-patient a known contact of a prior source case-patient?"
```

Comment:
This works for this field, but we'll need to look into passing in a JSX element instead of a string so we can do things like bold parts of the label. Good for this PR though.